addon/components/fixed-table-columns.js

```diff
@@ -1,11 +1,43 @@
+import Ember from 'ember';
 import layout from '../templates/components/table-columns';
 import TableColumns from './table-columns';
 
+const { computed } = Ember;
+
 export default TableColumns.extend({
-  layout: layout,
+  layout,
 
   init() {
     this._super(...arguments);
     this.classNames = ['fixed-table-columns'];
+  },
+
+  columnWidths: computed.mapBy('columns', 'width'),
+  columnBorderWidths: computed('columns.[]', function columnBorderWidths() {
```

Comment:
This is something I took from watching the commits on the ember-cli project. This mimics what the ES6 syntax does:

```
{
  foo() { ... }
}
```

compiles to this:

```
{
  foo: function foo() { ... }
}
```

The reason this becomes a useful thing to do is if an error occurs in the function, the console will display `columnBorderWidths` in the trace in place of `anonymous function` as it's no longer an anonymous function.